internal/http/services/sciencemesh/token.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/cs3org/reva/pkg/smtpclient"
 )
 
-const defaultInviteLink = "{{.MeshDirectoryURL}}?token={{.Token}}&providerDomain={{.User.Id.Idp}}"
+const defaultInviteLinkPrefix = "{{.MeshDirectoryURL}}?token={{.Token}}&providerDomain="
```

Comment:
Actually we can remove this template and simplify how we generate this link, to just a string concatenations.

Reply:
you mean end up with something like `meshURL + "?token=" + token + "&providerDomain=" + provider` ?

Reply:
Exactly

Reply:
this can be done here https://github.com/cs3org/reva/blob/580a83f0ae0b19e12e25bed057d9b977733a9c25/internal/http/services/sciencemesh/token.go#L137